### DISCUSSION TITLE
Problem mit Räumen in Verbindung mit Philips Hue

### DISCUSSION THREAD

**Discussion by detLAN** [thkl/hap-homematic#292] [Q&A]
Wenn ich in der Hue App eine Lampe einem Raum zuweise bzw. aus einem Raum entferne, dann werden alle Homematic Geräte aus dem entsprechenden Raum in den Standardraum verschoben.

Kann jemand dieses Verhalten Bestätigen / Reproduzieren und hat vielleicht eine Idee woran dies liegt?

**Comment by detLAN** [comment]
Ich habe diesbezüglich jetzt noch etwas festgestellt. Beim oben geschildertem Problem wird die entsprechende Instanz dem Standardraum zugeordnet und somit die Geräte entsprechend verschoben. Korrigiere ich die Instanz wieder, bleiben die Geräte jedoch weiterhin im Standard Raum. Kann @thkl vielleicht dazu etwas sagen?

**Comment by jb-home** [comment]
Die Hue App fummelt da leider zu viel in HomeKit rum. Daran kann nur Philips was ändern.
Das ist uns auch schon aufgefallen, wenn die Räume in Hue und HomeKit nicht Synchron sind. Dann beschafft die Hue App da einem einen verschiebe Spaß. Also so wenig wie möglich in der Hue App rum fummeln... Da kann @thkl auch nichts dran ändern.

**Reply by detLAN** [reply]
Danke für die Info, ich habe es ja schon vermutet. Das Problem ist nur, dass dies ja auch passiert, wenn man eine neue Lampe einbindet. Aber okay, dann müssen wir damit leben.
